tests/async-tokio.rs

```diff
@@ -0,0 +1,20 @@
+use quick_xml::events::Event::*;
+use quick_xml::Reader;
+
+#[tokio::test]
+async fn test_sample() {
+    let src = include_str!("documents/sample_rss.xml");
```

Comment:
This example should probably open a file instead of working with a `&str`

Reply:
We do not do that for `BufRead` implementation, so it is unclear why we should do that in that case

Reply:
Sorry, I think I though this test was an example last night.  It's fine to do this in tests.